hbase-server/src/test/java/org/apache/hadoop/hbase/wal/TestWALSplitValueCompression.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hbase.wal;
```

Comment:
Should be in `o.a.h.h.regionserver.wal` ?

Reply:
This is in the same package as `TestWALSplit` and five other similarly named units. Do you want me to move all of them?

Reply:
I don't know this history of this package, and especially since we now use a region from inside of the master, i don't know which package is better. I strongly prefer we keep related classes confined to a common package and clamp down accessors as much as we can ; he helps maintain hygiene throughout the codebase.

I feel strongly in that it should be done eventually. I don't feel strongly about you addressing this issue in this specific change. I guess your purpose is not to improve the codebase, but rather to implement this feature, a position to which I am sympathetic.